examples/ach/credit_external_bank/micro_deposits_test.go

```diff
@@ -112,7 +113,7 @@ func TestMicroDepositExample(t *testing.T) {
 		},
 		// not required since ACH is processed in batches,
 		// but useful in getting the full transfer model
-		moov.WithTransferWaitForRailResponse(),
+		moov.withTransferWaitFor(),
```

Comment:
I don't know if I agree with chopping that off. We can do `Wait-For`'s in more places than just the rail response.

Reply:
What's the reason to rename these? It's a breaking change for a minor rename? I could see `WithWaitForRailResponse()` as the underlying value is `rail-response`.

Reply:
Yeah, ignore the name change right now -- purpose here was to privatize the method since we don't need to expose this option if there's separate methods for creating sync vs async